### PR TITLE
Eri9600/open scene

### PR DESCRIPTION
**Pull request by eri9600**
@TADraeseke Open Scene with a portal item sample, shown here: https://devtopia.esri.com/runtime/common-samples/issues/566

### CODE REVIEW

File: java/open-scene-portal-item/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Open Scene - Portal Item
+Open a scene from a Portal item. These scenes can be stored in ArcGIS Online or Portal.
+
+![Open Scene Portal Item App](open-scene-portal-item.png)
+
+# How to use the sample
```
Comment (TADraeseke) [inline java/open-scene-portal-item/README.md]: ' ## '

File: java/open-scene-portal-item/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+When the sample loads, a scene from ArcGIS Online will be displayed. 
+
+# How it works
```
Comment (TADraeseke) [inline java/open-scene-portal-item/README.md]: ' ## '

File: java/open-scene-portal-item/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+This sample uses the scene portal item constructor to open a scene from ArcGIS Online. 
+
+#Relevant API
```
Comment (TADraeseke) [inline java/open-scene-portal-item/README.md]: ' ## '

File: java/open-scene-portal-item/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+* `PortalItem`
+* `Portal`
+* `ArcGISScene`
+
+#### Tags
+Scenes
```
Comment (TADraeseke) [inline java/open-scene-portal-item/README.md]: Replace 'Scenes' with 'Maps and Scenes' (one of our sample categories)

**Comment by eri9600** [comment]
@TADraeseke I have made the suggested changes. Let me know if everything is good.

File: java/open-scene-portal-item/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="app_name">Open Scene</string>
```
Comment (TADraeseke) [inline java/open-scene-portal-item/src/main/res/values/strings.xml]: Open Scene Portal Item

File: java/open-scene-portal-item/src/main/java/com/esri/arcgisruntime/sample/opensceneportalitem/MainActivity.java [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+
+    // inflate SceneView from layout
+    mSceneView= findViewById(R.id.sceneView);
+
+    // get the portal url and portal item from ArcGIS online
+    Portal portal = new Portal(getResources().getString(R.string.portal_url), false);
+    PortalItem portalItem = new PortalItem(portal, "a13c3c3540144967bc933cb5e498b8e4");
```
Comment (TADraeseke) [inline java/open-scene-portal-item/src/main/java/com/esri/arcgisruntime/sample/opensceneportalitem/MainActivity.java]: can extract string to item_id

File: java/open-scene-portal-item/src/main/java/com/esri/arcgisruntime/sample/opensceneportalitem/MainActivity.java [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.esri.arcgisruntime.sample.opensceneportalitem;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
```
Comment (TADraeseke) [inline java/open-scene-portal-item/src/main/java/com/esri/arcgisruntime/sample/opensceneportalitem/MainActivity.java]: add space between android and runtime imports

**Comment by eri9600** [comment]
@TADraeseke Added the comments you suggested. Merging now.

**Pull request merged**